src/app/config/config.service.ts

```diff
@@ -20,7 +20,7 @@ import { Injectable } from '@angular/core';
 @Injectable()
 export class ConfigService {
 
-  version = '2.0.0';
+  version = '2.0.14';
```

Comment:
Ignore that, I've only bumped up the version to test my references, because whenever there is a newer version `save()` is being called on the entity mapper to store the new version to the user document. We can decide the version handling when we start releasing, as we should have different configs for releases anyways.